transactions/src/main/resources/schema/wildfly-txn_5_1.xsd

```diff
@@ -0,0 +1,339 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
```

Comment:
Delete this file

Reply:
@kabir I really don't need this file when I do define a new schema version? I wonder first as I expected this one will be necessary and as a second point I fail with test `AbstractSubsystemBaseTest#testSchema` (I'm with investigation on it)

Reply:
@ochaloup You do not need a new schema version. Nothing has changed in the xml parsing, only in the management model. Sorry if I was not clear

Reply:
@kabir thanks Kabir. It's rather my trouble as I'm still not well familiar with the dependencies needed for upgrade.